test/unit/deprecation.test.ts

```diff
@@ -1,45 +1,3 @@
-import { readFileSync } from "fs";
-import { sign } from "@octokit/webhooks-methods";
-
-import { Webhooks } from "../../src";
-
-const pushEventPayloadString = readFileSync(
-  "test/fixtures/push-payload.json",
-  "utf-8"
-);
 describe("Deprecations", () => {
-  test("webhooks.verify(payload, signature) with object payload", async () => {
-    const spy = jest.spyOn(console, "error");
-    const secret = "mysecret";
-    const webhooks = new Webhooks({ secret });
-
-    await webhooks.verify(
-      JSON.parse(pushEventPayloadString),
-      await sign({ secret, algorithm: "sha256" }, pushEventPayloadString)
-    );
-    expect(spy).toBeCalledWith(
-      "[@octokit/webhooks] Passing a JSON payload object to `verify()` is deprecated and the functionality will be removed in a future release of `@octokit/webhooks`"
-    );
-    spy.mockClear();
-  });
-
-  test("webhooks.verifyAndReceive(payload, signature) with object payload", async () => {
-    const spy = jest.spyOn(console, "error");
-    const secret = "mysecret";
-    const webhooks = new Webhooks({ secret });
-
-    await webhooks.verifyAndReceive({
-      id: "123e456",
-      name: "push",
-      payload: JSON.parse(pushEventPayloadString),
-      signature: await sign(
-        { secret, algorithm: "sha256" },
-        pushEventPayloadString
-      ),
-    });
-    expect(spy).toBeCalledWith(
-      "[@octokit/webhooks] Passing a JSON payload object to `verifyAndReceive()` is deprecated and the functionality will be removed in a future release of `@octokit/webhooks`"
-    );
-    spy.mockClear();
-  });
+  test("there are currently no deprecations", () => {});
```

Comment:
Ehh empty tests are generally considered a mistake in jest - I think it'd be better to just delete this file

Reply:
I don't mind, this is just how the file was when we last did a major release removing deprecated methods